site-src/guides/index.md

```diff
@@ -296,8 +296,40 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
       kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/kgateway/httproute.yaml --ignore-not-found
       ```
 
-   1. Uninstall the CRDs
+   1. Uninstall the Gateway API Inference Extension CRDs
 
       ```bash
       kubectl delete -k https://github.com/kubernetes-sigs/gateway-api-inference-extension/config/crd --ignore-not-found
       ```
+
+   1. Choose one of the following options to cleanup the Inference Gateway.
+
+=== "GKE"
+
+      **TODO**
+
+=== "Istio"
+
+      **TODO**
+
+=== "Kgateway"
+
+      The following instructions assume you would like to cleanup ALL Kgateway resources that were created in this quickstart guide.
```

Comment:
nit: remove this line which repeats the exact same words as the cleanup starting phrase.

Reply:
I'm going to leave the line as-is for now. It can be removed in the future if we feel it's an issue.